service/models.py

```diff
@@ -231,6 +232,7 @@ def deserialize(self, data: dict):
         try:
             self.name = data["name"]
             self.user_id = data["user_id"]
+            self.status = data["status"]
```

Comment:
I was testing the API. User shouldn't have the need to pass this as a part of the json request, as it will always be the same as before, true by default. If someone wants to update the status, they call a different ACTION api whose job is to only manage this.